core/admin/start.py

```diff
@@ -19,7 +19,7 @@
     os.system("flask mailu admin %s %s '%s' --mode %s" % (account, domain, password, mode))
 
 start_command="".join([
-    "gunicorn -w 4 -b :80 ",
+    "gunicorn -w 4 -b [::]:80 ",
```

Comment:
it should be made conditional on SUBNET6 ... it will break on setups where ipv6 is disabled.

Getting ipv6 to work properly is more complicated than it looks: https://mailu.io/1.8/faq.html#how-to-make-ipv6-work

It may be simpler to force the resolution of IPv4 addresses as suggested in https://github.com/Mailu/Mailu/issues/1789#issuecomment-796538821